src/app/services/vm_service/vm_pool.ts

```diff
@@ -0,0 +1,54 @@
+export class VMPool {
+  private _reservations: {};
+  private _id: number;
+  private _shortName: string;
+  private _displayName: string;
+  private _maximumCount: number;
```

Comment:
to rozumiem jest machinesCount?

Reply:
wzięte prosto z backendu - w sumie może się wydawać że to jest to samo, ale machines count na dany termin może być mniejsza niż maximumCount (jeżeli przymiemy, że machinesCount jest liczbą dostępnych maszyn)

Reply:
okej to pewnie jest do obgadania, żeby na backendzie dorzucić jeszcze jakiś machinesCount na backu wtedy będzie to odrazu widać

Reply:
żeby dało się dorzucić machines count trzeba będzie podać datę, dla której jest to sprawdzane, także w tym wypadku nie ma takiej potrzeby